input/finisher-acquisition/bls.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "bls",
   "contents": [
-    // Halloween,
-    17,
-    // General,
-    27,45,51,10,28,39,34,46,44,16,57,59,14,29,13,15,24,25,12,43
+    // Living World Season 1: Scarlet's War
+    26,
+    // Gem Store
+    10,12,13,14,15,16,17,24,25,27,28,29,34,38,39,43,44,45,46,51,57,59
```

Comment:
I probably can set these up to be crawled from the wiki rather than manually maintained (I gather Guild Shield Finisher and one more was missing?)

Reply:
On further inspection this isn't really possible.

Reply:
Kinda unrelated, but you could apply the screenshot crawl method for the Black Lion Exclusives Chest, since it does have a preview option: https://wiki.guildwars2.com/wiki/Black_Lion_Exclusives_Chest

Not sure how often it gets updated though, and that's just for the container, it wouldn't cover those still not inside.

Reply:
I could, the big difficulty is always just in knowing when to check for changes... looks like traditionally end of August except this year there was a December update and last year there was an April update too.

Reply:
Manually then, I'm tracking skins myself anyway, so it's no effort to update this now and then, at least my work will be useful for others :D.